### PR TITLE
Asp.net 5 renames

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Putting this out now while we wait for the rest of the builds to settle down.

/cc @pakrym @troydai @JunTaoLuo

### CODE REVIEW

File: test/Microbenchmarks.Tests/project.json [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
+    "Microsoft.AspNetCore.TestHost": "1.0.0-*",
+    "Microsoft.AspNetCore.Tooling.Razor": "1.0.0-*"
   },
   "commands": {
     "test": "xunit.runner.aspnet"
   },
   "frameworks": {
-    "dnxcore50": { },
-    "dnx451": { }
+    "dnxcore50": {},
```
Comment (JunTaoLuo) [inline test/Microbenchmarks.Tests/project.json]: I'm assuming the TFM's are going to be updated in a separate PR?

**Comment by troydai** [comment]
The changes look good, but what's the deal with all the first line changes? Are these BOM changes?

**Comment by NTaylorMullen** [comment]
@troydai Ya, they had BOMs before, now they dont. Script magic :smile:

File: test/Microbenchmarks.Tests/project.json (hunk 1 above)
Comment (NTaylorMullen) [inline test/Microbenchmarks.Tests/project.json]: Separate PR. Myget doesn't support the new TFMs yet :smile:

**Comment by troydai** [comment]
Cool. :shipit:
